tests-ng/handlers/configure_nvme.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from plugins.shell import ShellRunner
+import json
+
+# Define variables for the IP address, NVMe device, and subsystem name
+IP_ADDRESS="127.0.0.1"
+NVME_DEVICE="/tmp/nvme.img"
+SUBSYSTEM_NAME="testnqn"
+PORT_NUMBER="4420"
+TRTYPE="tcp"
+ADRFAM="ipv4"
+
+@pytest.fixture
+def nvme_device(shell: ShellRunner):
+    shell(f"truncate -s 512M {NVME_DEVICE}")
+    shell("DEBIAN_FRONTEND=noninteractive apt-get install -y mount")
+    shell(f"losetup -fP {NVME_DEVICE}")
+
+    # Load necessary kernel modules
+    shell("modprobe nvme_tcp")
+    shell("modprobe nvmet")
+    shell("modprobe nvmet-tcp")
+
+    # Create the NVMe subsystem directory
+    shell(f"mkdir -p /sys/kernel/config/nvmet/subsystems/{SUBSYSTEM_NAME}")
+
+    # Set the subsystem to accept any host
+    shell(f"echo 1 | sudo tee /sys/kernel/config/nvmet/subsystems/{SUBSYSTEM_NAME}/attr_allow_any_host")
+
+    # Create and configure the namespace
+    shell(f"mkdir -p /sys/kernel/config/nvmet/subsystems/{SUBSYSTEM_NAME}/namespaces/1")
+    shell(f"echo -n {NVME_DEVICE} | tee /sys/kernel/config/nvmet/subsystems/{SUBSYSTEM_NAME}/namespaces/1/device_path")
+    shell(f"echo 1 | tee /sys/kernel/config/nvmet/subsystems/{SUBSYSTEM_NAME}/namespaces/1/enable")
+
+    # Configure the NVMe-oF TCP port
+    shell("mkdir -p /sys/kernel/config/nvmet/ports/1")
```

Comment:
What happens if port 1 already exists? Maybe it does make sense to look for existing ports and choose the next free one?